ros/src/pybullet_ros/plugins/body_vel_control.py

```diff
@@ -60,20 +61,20 @@ def lookupTransform(self, target_frame='base_link', source_frame='odom'):
         source_frame = odom, world_frame, target_frame = base_link, robot_frame
         """
         # get robot pose from pybullet
-        t, r = self.pb.getBasePositionAndOrientation(self.robot)
+        t, r, _, _ = self.robot.get_base_state()
```

Comment:
as a tip, you can use either of the following options here to help with unpacking:
```suggestion
        t, r, *_ = self.robot.get_base_state()
        t, r = self.robot.get_base_state()[0:2]
```

Reply:
Ah yeah nice, thanks